script.js

```diff
@@ -18,21 +25,59 @@ if (data && data.length > 0) {
 }
 
 function AddTodo(todoText) {
-    const element = document.createElement("LI")
-    const elementText = document.createTextNode(todoText)
-    element.appendChild(elementText)
+        const element = document.createElement("LI");
+	const elementBtn = document.createElement("button");
+	const elementText = document.createTextNode(todoText);
+	var elementId = nameId+loop;
+	loop++;
+	elementBtn.id = elementId;
+	element.id = elementId;
+	elementBtn.type = "submit";
+	elementBtn.innerHTML = "x";
+	elementBtn.addEventListener("click",function(){ removeHandler(elementBtn.id); });
 
+	element.appendChild(elementText);
+	element.appendChild(elementBtn);
+
+	
     // Update the View
-    outputListElement.appendChild(element)
+	outputListElement.appendChild(element);
 
     // Update the Model
-    todoCollection.push(todoText);
-    localStorage.setItem(localStorageKey, JSON.stringify(todoCollection))
+        todoCollection.push(todoText);
+	localStorage.setItem(localStorageKey, JSON.stringify(todoCollection));
+	
+}
+function RemTodo(todoText) {
+	
+	  item = document.getElementById(todoText);
+	  outputListElement.removeChild(item);
```

Comment:
this only removes an element from the List in the User Interface but the localStorage Database still holds this element. So when you reload the page, the todo app still holds the deleted item.

Reply:
Please make sure you safe the current state of the todos in the local storage
Read more about local Storage: https://blog.logrocket.com/the-complete-guide-to-using-localstorage-in-javascript-apps-ba44edb53a36/